pkg/cloud/services/gc/ec2.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gc
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+func (s *Service) deleteSecurityGroups(ctx context.Context, resources []*AWSResource) error {
+	for _, resource := range resources {
+		if !s.isSecurityGroupToDelete(resource) {
+			s.scope.V(4).Info("Resource not a security group for deletion", "arn", resource.ARN.String())
```

Comment:
I wonder if this is relevant info here to display. As things are implemented, you can, by design, call this function with all of the resources and it would basically just filter resources that it needs. I think this log message will be just distraction when, by design, it can encounter other resources which it will just skip.

Reply:
I agree that it could be a distraction and that's why I added the logs entries with verbosity of 4 & 5. If you are debugging a problem these will be helpful as you can see it was skipped and the arn.

Reply:
Changed so they are all trace level (5).